directives/questionnaire.py

```diff
@@ -361,6 +366,9 @@ def input_type(self):
 
 class MultipleChoice(Choice):
     ''' Lists options for picking all the correct ones. '''
+    # inherit QuestionMixin options and add a key
+    option_spec = dict(QuestionMixin.option_spec)
+    option_spec.update({'partial-points': directives.unchanged})
```

Comment:
Ǹo need to use the `update` method for a single key. Do it like this: 
`option_spec['partial-points'] = directives.flag`

This is supposed to be a flag, right? So use `directives.flag`.